### PR TITLE
Order Cycle design sweep

### PR DESCRIPTION
**Pull request by luisramos0**
#### What? Why?

This PR addresses most points in #4572

OC Selector

- [x] Increase width of OC selector dropdown so more characters can fit in [desktop & tablet]
               - the width is currently 200px, in this PR we make it 250px for screens between 640px and 1024px and screens larger than 1200px

- [x] Widen OC selector dropdown on RIGHT SIDE so it has equal margins on left and right sides [mobile]
- [x] Add padding to the left of the dropdown caret/arrow so it’s not positioned directly next to the text [all breakpoints]

Both points above:
image

No open OC

- [x] Exclamation point ‘!’ and ‘Orders are closed’ text is not vertically centred with the shape [mobile]
image

#### What should we test?
Verify points described above.

#### Release notes
Changelog Category: Changed
Improved design details according to Order Cycle design sweep.

### CODE REVIEW

File: app/assets/stylesheets/darkswarm/_shop-navigation.css.scss [hunk 1]
```diff
@@ -61,6 +61,8 @@ ordercycle {
 
     select {
       background-image: url('/assets/white-caret.svg');
+      background-size: 30px auto;
+      background-position-x: 102%;
```
Comment (luisramos0) [inline app/assets/stylesheets/darkswarm/_shop-navigation.css.scss:65]: this shifts the caret a little to the right.

File: app/assets/stylesheets/darkswarm/_shop-navigation.css.scss [hunk 2]
```diff
@@ -69,23 +71,26 @@ ordercycle {
...
-      border-radius: 0 0.25em 0.25em 0;
       min-width: 13em;
+      width: 200px;
 
       @media all and (max-width: 480px) {
         width: 100%;
       }
+
+      @media all and (min-width: 640px) and (max-width: 1024px), (min-width: 1200px) {
+        width: 250px;
```
Comment (luisramos0) [inline app/assets/stylesheets/darkswarm/_shop-navigation.css.scss:92]: there's no space for this in screens between 1024px and 1200px

File: app/assets/stylesheets/darkswarm/_shop-navigation.css.scss [hunk 3]
```diff
@@ -69,23 +71,26 @@ ordercycle {
 
     select,
     p {
-      width: inherit;
       display: inline-block;
       color: $white;
       background-color: transparent;
       border: 0;
-      margin-bottom: 0;
+      border-radius: 0 0.25em 0.25em 0;
```
Comment (sauloperez) [inline app/assets/stylesheets/darkswarm/_shop-navigation.css.scss:78]: you can reuse the var `$radius-small` here defined in https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/assets/stylesheets/darkswarm/variables.css.scss#L36

**Comment by sauloperez** [comment]
> - the width is currently 200px, in this PR we make it 250px for screens between 640px and 1024px and screens larger than 1200px

why not that width for those screens between 1024px and 1200px? if it fits in smaller screens why doesn't it for those a bit bigger?

File: app/assets/stylesheets/darkswarm/_shop-navigation.css.scss [hunk 4]
```diff
@@ -69,23 +71,26 @@ ordercycle {
...
-      background-size: 30px auto;
-      border-radius: 0 0.25em 0.25em 0;
       min-width: 13em;
+      width: 200px;
 
       @media all and (max-width: 480px) {
         width: 100%;
       }
+
+      @media all and (min-width: 640px) and (max-width: 1024px), (min-width: 1200px) {
```
Comment (sauloperez) [inline app/assets/stylesheets/darkswarm/_shop-navigation.css.scss:91]: > - the width is currently 200px, in this PR we make it 250px for screens between 640px and 1024px and screens larger than 1200px

why not that width for those screens between 1024px and 1200px? if it fits in smaller screens why doesn't it for those a bit bigger? Unless there's something obvious I'm not seeing to me that should from 640px onwards like

```
@media all and (min-width: 640px)  {
  width: 250px;
}
```

Comment (luisramos0) [reply app/assets/stylesheets/darkswarm/_shop-navigation.css.scss:91]: for screens smaller than 1024px the OC selector is an horizontal bar. above 1024px it moves to the top right where it doesnt have space for this until the screen gets to 1200px

File: app/assets/stylesheets/darkswarm/_shop-navigation.css.scss (hunk 3 above)
Comment (luisramos0) [reply app/assets/stylesheets/darkswarm/_shop-navigation.css.scss]: I rebased this PR to resolve conflicts and this line was exactly where the conflict was and guess what the line in master looked like? same but with $radius-small :-)

**Comment by sauloperez** [comment]
Code Climate is giving feedback worth checking

**Comment by luisramos0** [comment]
ouch, I messed up while rebasing/resolving conflicts.
fixed now.

**Comment by sauloperez** [comment]
This hasn't been reviewed by anyone else in the last two weeks, so before risking this branch becoming stale, I'm moving it forward.

**Comment by filipefurtad0** [comment]
Hi @luisramos0 ,
This looks much better now.

Things to check:
1) Increase width of OC selector dropdown [desktop & tablet] 200px -> 250px
2) Widen OC selector dropdown on RIGHT SIDE
3) Add padding to the left of the dropdown caret/arrow
4) Vertically centre point ‘!’ and ‘Orders are closed’ text

On points 1) 2 and 3):

- Desktop

before staging this PR: 208 px
image


after staging this PR: 250 px width - points 1) 2) and 3) are visible:
image


- Mobile

Before the PR:
image

After the PR:
image


On point 4)

Mobile:

before staging this PR: 
image


after staging this PR:
image

Ready to go!
Pinging @daniellemoorhead - would you add any other check to this PR?

**Comment by daniellemoorhead** [comment]
>Pinging @daniellemoorhead - would you add any other check to this PR?

These were @yukoosawa's sweep so will leave her to review your most excellent testing notes and make sure she's happy with it 😃

**Comment by yukoosawa** [comment]
@daniellemoorhead @luisramos0 @filipefurtad0 testing notes + screenshots look good! ≈ 30 characters for the OC selector is great!!

**Comment by daniellemoorhead** [comment]
@yukoosawa just wanted to check whether this means all the items in your Order Cycle design sweep document are now done and therefore we could close the OC epic?

**Pull request merged**